packages/cli/src/util/client.ts

```diff
@@ -31,18 +29,22 @@ export async function createMedplumClient(options: MedplumClientOptions, profile
 }
 
 function getClientValues(options: MedplumClientOptions, storage: FileSystemStorage): Record<string, any> {
```

Comment:
Is the return type of this method `MedplumClientOptions`?  Let's try to use types if they're available.